src/components/Books/Books.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+
+function Books() {
+  return (
+    <>
+      <div className="d-flex justify-content-center mt-5">
+        <form>
+          <h2>Add a Book</h2>
```

Comment:
- [ ] Please as it is a requirement for this project and best practice; making sure the building blocks of your app are set as re-usable components, i.e. each components should perform a single task(e.g. `rendering a book`, `creating a new book`). Please it will be nice to add the following components to your app and render them in your `Book.js` component;
- `Book` : Takes the book data as a prop and renders the `title` and the `author`, Has a button for removing books(the button wouldn't remove the books yet, this is just a placeholder for now).
- `CreateNewBook`: Renders a form with inputs for the title and the author of the book, Has a button to submit(the button wouldn't add the book yet, this is just a placeholder for now).

See screenshots of the requirements below for more clarifications.

![buildingBlocks](https://user-images.githubusercontent.com/65852621/150279259-26838858-cf7f-402a-ba7a-971e1c30b9b2.png)

![BuildingBlocks2](https://user-images.githubusercontent.com/65852621/150279290-98e2eb29-eb79-4fb3-ac67-75bbe7787db8.png)